runners/core-java/src/main/java/org/apache/beam/runners/core/SplittableParDo.java

```diff
@@ -78,27 +90,92 @@
  * ParDo.of(splittable DoFn)}, but not for direct use by pipeline writers.
  */
 @Experimental(Experimental.Kind.SPLITTABLE_DO_FN)
-public class SplittableParDo<
-        InputT, OutputT, RestrictionT, TrackerT extends RestrictionTracker<RestrictionT>>
-    extends PTransform<PCollection<InputT>, PCollection<OutputT>> {
-  private final DoFn<InputT, OutputT> fn;
-  private final DoFnSignature signature;
+public class SplittableParDo {
+  /** The version of {@link SplittableParDo} for a {@link ParDo} with a single main output. */
+  public static class Bound<InputT, OutputT, RestrictionT>
```

Comment:
[BEAM-447](https://issues.apache.org/jira/browse/BEAM-447) "stop referring to types as `Bound`/`Unbound`".

Especially here where there is no such duality.